buildtest/cli/history.py

```diff
@@ -103,21 +104,18 @@ def list_build_history(no_header=None, terse=None, pager=None, color=None):
         table["fail_rate"].append(content["test_summary"]["fail_rate"])
 
     if terse:
-        row_entry = []
-
-        for key in table.keys():
-            row_entry.append(table[key])
-
+        row_entry = [table[key] for key in table.keys()]
         transpose_list = [list(i) for i in zip(*row_entry)]
 
-        # We print the table columns if --no-header is not specified
-        if not no_header:
-            console.print("|".join(table.keys()), style=consoleColor)
+        with console.pager() if pager else contextlib.suppress():
```

Comment:
i dont like the use of another library this is a bit cryptic. Can we remove the `console.print` statement, instead create a variable and later do something like

```
if pager:
  with console.pager():
      console.print(...)  
```

Reply:
take a look at something like this https://github.com/buildtesters/buildtest/blob/f9e08dd2fd77395e59adfc8bb4f3948c36485fef/buildtest/cli/buildspec.py#L1203-L1206. Basically you can do pager with any output provided you build the variable or method with the necessary print statements.

Reply:
@shahzebsiddiqui Okay got it. I have create a function now for printing in terse mode.